tests/testenv.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+usage()
+{
+	echo "$0 [arch: x86/x64/arm/arm64] [flavor: debug/release]"
+	exit 1
```

Comment:
What is the idea behind the testenv.sh script? It is just used from a single place, runtest.sh does similar parsing - and ignores what is set.

Is it expected that the `CoreRT_BuildArch/CoreRT_BuildType` variables are be set in your shell environment, or is it just something that the different scripts use to communicate? If it is the later, it would be nice to use explicit arguments instead.